Assets/ConsentManagementProvider/Plugins/iOS/Source/SwiftBridge.swift

```diff
@@ -0,0 +1,334 @@
+//
+//  SwiftBridge.swift
```

Comment:
Hey @andresilveirah check this new almost-pure Swift Bridge I've built for the Unity SDK!